### ISSUE TITLE
Make `icaruscode` depend on `sbncode`.

### ISSUE THREAD

**Issue by PetrilloAtWork** [SBNSoftware/icaruscode#11]
`icaruscode` is going to depend on `sbncode` (and indirectly on `sbnobj`).

**Comment by PetrilloAtWork** [comment]
Depends on SBNSoftware/sbncode#6.

**Comment by PetrilloAtWork** [comment]
Branch SBNSoftware/icaruscode#feature/repositoryRestructuring now contains a version of `icaruscode` that is compatible with `sbncode` branch SBNSoftware/sbncode#feature/repositoryRestructuring. Similar branches are available for SBNSoftware/sbnobj and SBNSoftware/sbndcode.
How does GitHub references work here? Obviously I don't understand it yet.

Anyway: the adaptation work has been much less than trivial for the CRT data, which got merged, split and changed at the same time. I have started a document to be kept in `sbncode` repository to document all changes in one place.
When the actual merge ends, most of this work will need to be repeated if any change in the moved code has occurred in the meanwhile.
